t/cli/test_admin.sh

```diff
@@ -154,6 +154,43 @@ fi
 
 echo "pass: missing admin key and show ERROR message"
 
+# missing admin key, only allow 127.0.0.0/24 to access admin api
+
+echo '
+deployment:
+  admin:
+    allow_admin:
+      - 127.0.0.0/24
+    allow_admin: ~
+' > conf/config.yaml
+
+make init > output.log 2>&1 | true
+
+grep -E "ERROR: missing valid Admin API token." output.log > /dev/null
+if [ $? -eq 0 ]; then
```

Comment:
The condition here is still incorrect, please think again what is going to check in this test.

Reply:
I have checked with @spacewander, the condition is correct

Reply:
@monkeyDluffy6017 
Yes, you are right. Sorry for the wrong suggestion.